### PR TITLE
Minor Quickstart revisions

### PR DESCRIPTION
**Pull request by lazeratops**
This PR:

* Adds a placeholder README (will be expanded with a link to the Quickstart guide once that is live)
* Logs error on call leave (if any) as suggested in Quickstart guide review
* Retrieves the layout once on call init as opposed to for each participant join as suggested in Quickstart guide review
* Reformats a comment and removes some unneeded blank lines

### CODE REVIEW

File: app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt [hunk 1]
```diff
@@ -81,15 +78,17 @@ class MainActivity : AppCompatActivity() {
...
-
-
         })
 
         findViewById<Button>(R.id.leave)
             .setOnClickListener {
                 Log.d("BUTTONS", "User tapped the Leave button")
                 call.leave {
-                    Log.d("Call", "Call has been left")
+                    if (it.isError) {
```
Comment (lazeratops) [inline app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt:87]: LMK if there's a better "Kotlin-y" way to do this, I just went with what made most sense to me after looking at the return object.

File: app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt [hunk 2]
```diff
@@ -81,15 +78,15 @@ class MainActivity : AppCompatActivity() {
...
-
-
         })
 
         findViewById<Button>(R.id.leave)
             .setOnClickListener {
                 Log.d("BUTTONS", "User tapped the Leave button")
                 call.leave {
-                    Log.d("Call", "Call has been left")
+                    it.error?.apply {
```
Comment (lazeratops) [inline app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt:87]: LMK if there's a more idiomatic "Kotlin-y" way to do this, I just went with what made most sense to me after looking at the return object.

Comment (marcus-daily) [reply app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt:87]: This looks good to me!

**Review by marcus-daily — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**